src/commands/url.js

```diff
@@ -30,4 +46,21 @@ export default program => {
     .allowOffline()
     .allowNonInteractive()
     .asyncActionProjectDir(action);
+
+  program
+    .command('url:ipa [project-dir]')
+    .alias('ui')
```

Comment:
I know we use these short aliases in a bunch of commands (I'm not a big user of them myself), but I think `exp ui` might be confusing. Maybe let's just add `url:ipa` and `url:apk` for now and no aliases?